tests/regression/lp-2044335/task.yaml

```diff
@@ -23,8 +23,18 @@ prepare: |
     rm -f test-snapd-busybox-static*.snap
     snap connect test-snapd-busybox-static:system-packages-doc
 
+    if [ ! -e "/usr/share/doc/linux-image-$(uname -r)/changelog.Debian.gz" ]; then
```

Comment:
do we care about this particular file, or could we equally well test any other file that's highly likely to be present under /usr/share/doc (eg coreutils or bash?)